converter/fluka/cards/scoring_card.py

```diff
@@ -1,42 +1,112 @@
 from dataclasses import dataclass, field
+from typing import Optional, Union
 
 from converter.fluka.cards.card import Card
-from converter.fluka.helper_parsers.scoring_parser import Scoring
+from converter.fluka.helper_parsers.detector_parser import Detector
+from converter.fluka.helper_parsers.scoring_parser import CustomFilter, ParticleFilter, Quantity, Scoring
 
 
-def handle_first_card(scoring: Scoring, output_unit: int = 21) -> str:
-    """Creates first Scoring card"""
+@dataclass
+class ScoringCardIndexCounter:
+    """Class representing counter for Fluka scoring cards"""
```

Comment:
Why do you need a dedicated class here ? simple int would be enough

Reply:
We can have multiple scorings with multiple quantities, therefore we have to use wrapper (class).
Which will be shared by all scorings.
We are alos incrementing this counter inside of `handle_scoring_cards` function.
And sometimes we do not increment its value.

In future, it should be moved to class responsible for parsing all types of scorings etc.